File: py/prob_832.py
class Solution:
    def flipAndInvertImage(self, A):
        """
        :type A: List[List[int]]
        :rtype: List[List[int]]
        """
        N = len(A)
        for i in range(N):
            A[i] = A[i][::-1]
            for j in range(N):
                A[i][j] = 1 if A[i][j] == 0 else 0
        return A

File: cpp/prob_832.cpp
class Solution {
public:
    vector<vector<int>> flipAndInvertImage(vector<vector<int>>& A) {
        int N = A.size();
        for (int i=0; i<N; i++) {
            reverse(A[i].begin(),A[i].end());
            for (int j=0; j<N; j++) {
                A[i][j] = A[i][j] == 0 ? 1 : 0;
            };
        };
        return A;
    };
};